### PR TITLE
Update FCB datasheet link

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Updated Foundation Cloud Build datasheet link

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/cloud/foundation-cloud
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that the datasheet links take you to https://assets.ubuntu.com/v1/b71c0af7-canonical-foundation-cloud-build-2018-08-09.pdf

### CODE REVIEW

File: templates/cloud/foundation-cloud/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
...
+        <p><a href="{{ ASSET_SERVER_URL }}6e65055c-EN_Foundation-Cloud-Build_screen-AW_07.08.18.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read the datasheet', 'eventLabel' : 'Read the datasheet - Top CTA' , undefined });" class="p-button--positive">Read the datasheet</a>&emsp;<a href="/cloud/contact-us?product=foundation-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Talk to us - Top CTA' , undefined });" class="p-button--neutral">Talk to us</a></p>
```
Comment (pmahnke) [inline templates/cloud/foundation-cloud/index.html]: sorry, but can we rename this to Foundation-Cloud-Build-2018-08.pdf

**Comment by kasiag** [comment]
Can you replace current datasheet link with 
[EN_Foundation-Cloud-Build_screen-AW_09.08.18.pdf](https://github.com/canonical-websites/www.ubuntu.com/files/2274570/EN_Foundation-Cloud-Build_screen-AW_09.08.18.pdf)

**Comment by kasiag** [comment]
can you replace current link to the datasheet with 
[EN_Foundation-Cloud-Build_screen-AW_09.08.18.pdf](https://github.com/canonical-websites/www.ubuntu.com/files/2274573/EN_Foundation-Cloud-Build_screen-AW_09.08.18.pdf)

**Comment by Caleb-Ellis** [comment]
@pmahnke Updated with new file and filename

**Pull request merged**